proposals/021-jsr-45.md

```diff
@@ -0,0 +1,38 @@
+# Complete SCP-11: implement JSR-45 for improved Scala debugging experience
+
+## Proposer
+
+Proposed by Ólafur Páll Geirsson, Twitter, December 2nd 2019.
+
+## Abstract
+
+This is a proposal to prioritize the completion of
+[SCP-11](https://github.com/scalacenter/advisoryboard/blob/master/proposals/011-debugging-symbols.md).
+To reduce the scope of SCP-11, this proposal suggests to focus only contributing
+JSR-45 support to the Scala compiler.
+
+[JSR-45](https://jcp.org/en/jsr/detail?id=45) is an approved specification that
+establishes standardized tools for correlating Java byte code to source code of
+languages other than the Java programming language. For example, the Kotlin
+compiler uses JSR-45 to improve the developer experience when working with
+debugging tools and code coverage tools.
+
+## Proposal
+
+The end goal of this proposal is to fix issue
+[scala/scala-dev#3](https://github.com/scala/scala-dev/issues/3). The Scala
+Center is expected to collaborate with the Scala compiler team to ensure that
+the JSR-45 implementation is well tested, documented and comprehensive (handles
+cases like inlined functions).
+
+## Cost
+
+Six full-time engineer months to implement JSR-45 support in the compiler.
+
+On the condition that the initial implementation has been completed and
+released, additional six part-time engineering months for maintainenance and
+support.
+
+## Timescales
+
+It would be ideal to complete this project sometime within the year of 2019.
```

Comment:
This isn't realistic given the "six full-time months" point above.

Reply:
whoops, meant 2020

Reply:
BUT I WANT JSR-45 RIGHT NOW!! 😅